x/incentive/types/codec.go

```diff
@@ -18,9 +18,14 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 		(*sdk.Msg)(nil),
 		&MsgWithdrawReward{},
 		&MsgUpdateParams{},
+		&MsgSetWithdrawAddress{},
 	)
 
 	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
+
+	registry.RegisterImplementations((*sdk.Msg)(nil),
+		&MsgWithdrawReward{},
+	)
```

Comment:
don't think this is needed but we should reigister the type in `RegisterCodec` above`